logprep/util/configuration.py

```diff
@@ -239,6 +261,13 @@ def _perform_verfification_and_get_errors(
                 errors.append(error)
         return errors
 
+    def _verify_environment(self):
+        errors = []
+        for missing_var in self._getter.missing_env_vars:
+            errors.append(MissingEnvironmentError(missing_var))
+        if errors:
+            raise InvalidConfigurationErrors(errors)
```

Comment:
no... I want to have multiple errors, not one error with a joined string

Reply:
Well but then you should revert the error string again to singular form as it now will look like this if more than one env variable is missing:
```
Environment variable(s) used, but not set: VAR_ONE
Environment variable(s) used, but not set: VAR_TWO
Environment variable(s) used, but not set: VAR_THREE
```
Though I still think that something like this would look nicer:
```
Environment variable(s) used, but not set: VAR_ONE, VAR_TWO, VAR_THREE
```
If you insist on separated error messages then I will still approve it after you use the singular form.